docs/hip-7-changes.rst

```diff
@@ -0,0 +1,338 @@
+.. meta::
+  :description: This chapter provides an introduction to the HIP API.
+  :keywords: AMD, ROCm, HIP, CUDA, C++ language extensions
+
+.. _compatibility-changes:
+
+*******************************************************************************
+CUDA compatibility changes to HIP API
```

Comment:
The tittle should be for HIP APIs, like "Backwards Incompatible Changes in HIP API"